test/v6/ukPropertyBsas/retrieve/RetrieveUkPropertyBsasServiceSpec.scala

```diff
@@ -68,6 +68,17 @@ class RetrieveUkPropertyBsasServiceSpec extends ServiceSpec {
             await(service.retrieve(request)) shouldBe Left(ErrorWrapper(correlationId, RuleTypeOfBusinessIncorrectError))
           })
       }
+      "downstream returns a success response with invalid Tax Year" should {
```

Comment:
(And elsewhere in SE and Foreign.) Maybe be explicit about what's wrong with the tax year and that this is for pre-TYS requests:
```suggestion
      "downstream returns a  a Tax Year different from that in the pre-TYS request" should {
```